src/WarcraftLegacies.Source/Quests/Scourge/QuestDrakUnlock.cs

```diff
@@ -24,23 +24,22 @@ public sealed class QuestDrakUnlock : QuestData
     /// <summary>
     /// Initializes a new instance of the <see cref="QuestDrakUnlock"/> class.
     /// </summary>
-    public QuestDrakUnlock(Rectangle rescueRect, Capital draktharonKeep) : base(
+    public QuestDrakUnlock(Rectangle rescueRect) : base(
       "Drak'tharon Keep", "Drak'tharon Keep will be the perfect place for an outpost by the sea.",
       @"ReplaceableTextures\CommandButtons\BTNUndeadShipyard.blp")
     {
       AddObjective(new ObjectiveControlPoint(ControlPointManager.Instance.GetFromUnitType(Constants.UNIT_N030_DRAK_THARON_KEEP)));
-      AddObjective(new ObjectiveControlCapital(draktharonKeep, false));
       AddObjective(new ObjectiveExpire(660, Title));
       AddObjective(new ObjectiveSelfExists());
       ResearchId = Constants.UPGRADE_R08J_QUEST_COMPLETED_DRAK_THARON_KEEP;
       _rescueUnits = rescueRect.PrepareUnitsForRescue(RescuePreparationMode.HideNonStructures);
     }
 
     /// <inheritdoc/>
-    protected override string RewardFlavour => "Drak'tharon Keep is now under the control of the Scourge.";
+    protected override string RewardFlavour => "Drak'tharon Keep is now under the control of the Scourge and Kel'tuzad has joined the Scourge.";
 
     /// <inheritdoc/>
-    protected override string RewardDescription => "Control of all buildings in Drak'tharon Keep";
+    protected override string RewardDescription => "Control of all buildings in Drak'tharon Keep and can train Kel'tuzad at the altar of darkness";
```

Comment:
```suggestion
    protected override string RewardDescription => "Gain control of all buildings in Drak'tharon Keep and learn to train Kel'thuzad from the Altar of Darkness";
```
Again it would be better to use `GetObjectName` here.